### PR TITLE
Expanded sequence sorting

### LINKED ISSUE #866: Add log that says sequence commands are not sorted
This is just to add a metadata field to sequence telling if the commands are sorted or not.

### LINKED ISSUE #879: Time sort expansion commands only if there is more than one activity
We need to update our time sorting and only sort if there is more than one activity.

- If there is a mix of absolute and relative times (where each command has an absolute first command), make all times for both activities absolute, then sort. Return all times as absolute.
- Remove all single activity -> sequencing sorting

### PR DESCRIPTION
**Pull request by cohansen**
Closes #866 
Closes https://github.com/NASA-AMMOS/aerie/issues/879

* **Review:** By commit
* **Merge strategy:** Merge (no squash)

## Description
Adds a new `timeSorted` flag to the sequence metadata and adds sorting if every command is `EPOCH_RELATIVE`.

## Verification
Updated existing tests and added new tests.

## Documentation
NA

## Future work
Possibly more sorting behavior? Hopefully this catches all of the valid cases.

### CODE REVIEW

**Review by camargo — APPROVED** [review]
Tested locally, LGTM

File: sequencing-server/src/defaultSeqBuilder.ts [hunk 1]
```diff
@@ -7,80 +7,95 @@ export const defaultSeqBuilder: SeqBuilder = (
...
+  let previousTime: Temporal.Instant | null = null;
+
+  // If we have mixed time commands then we only sort if they're all absolute and relative.
   for (const command of commands) {
     // If the command is epoch-relative, complete, or the first command and relative then short circuit and don't try and sort.
     if (
       command.epochTime !== null ||
       (command.absoluteTime === null && command.epochTime === null && command.relativeTime === null) ||
       (command.relativeTime !== null && commands.indexOf(command) === 0)
     ) {
```
Comment (cohansen) [inline sequencing-server/src/defaultSeqBuilder.ts]: Okay I understand, you're correct in your assumption. I'll change it so it checks the first command of each activity.

**Review by cartermak — APPROVED** [review]
LGTM, thanks for taking the time to workshop the details!

**Comment by camargo** [comment]
Thanks so much for taking a look @cartermak, it's super helpful

**Pull request merged**